tests/Feature/JobControllerTest.php

```diff
@@ -173,42 +174,59 @@ public function testSubmitForReview() : void
         });
     }
 
-    /**
-     * Ensure a manager can view the create Job Poster form.
-     *
-     * @return void
-     */
-    public function testManagerCreateView() : void
-    {
-        $response = $this->actingAs($this->manager->user)
-        ->get('manager/jobs/create');
-        $response->assertStatus(200);
-
-        $response->assertSee(e(Lang::get('manager/job_create')['title']));
-        $response->assertViewIs('manager.job_create');
-
-        $response->assertSee(e(Lang::get('manager/job_create', [], 'en')['questions']['00']));
-        $response->assertSee(e(Lang::get('manager/job_create', [], 'fr')['questions']['00']));
-    }
+    //TODO: Managers cannot create job posters until Job Poster Builder is complete
+
+    // /**
+    //  * Ensure a manager can view the create Job Poster form.
+    //  *
+    //  * @return void
+    //  */
+    // public function testManagerCreateView() : void
+    // {
+    //     $response = $this->actingAs($this->manager->user)
+    //     ->get('manager/jobs/create');
+    //     $response->assertStatus(200);
+
+    //     $response->assertSee(e(Lang::get('manager/job_create')['title']));
+    //     $response->assertViewIs('manager.job_create');
+
+    //     $response->assertSee(e(Lang::get('manager/job_create', [], 'en')['questions']['00']));
+    //     $response->assertSee(e(Lang::get('manager/job_create', [], 'fr')['questions']['00']));
+    // }
+
+    // /**
+    //  * Ensure a manager can create a Job Poster.
+    //  *
+    //  * @return void
+    //  */
+    // public function testManagerCreate() : void
+    // {
+    //     $newJob = $this->generateEditJobFormData();
+
+    //     $dbValues = array_slice($newJob, 0, 8);
+
+    //     $response = $this->followingRedirects()
+    //     ->actingAs($this->manager->user)
+    //     ->post('manager/jobs/', $newJob);
+    //     $response->assertStatus(200);
+    //     $response->assertViewIs('applicant.job_post');
+    //     $this->assertDatabaseHas('job_posters', $dbValues);
+    //     $response->assertSee(e(Lang::get('applicant/job_post')['apply']['edit_link_title']));
+    // }
 
     /**
-     * Ensure a manager can create a Job Poster.
+     * Ensure an Admin can create a Job Poster for a manager.
      *
      * @return void
      */
-    public function testManagerCreate() : void
+    public function testCreateAsManager() : void
     {
-        $newJob = $this->generateEditJobFormData();
-
-        $dbValues = array_slice($newJob, 0, 8);
+        $admin = factory(User::class)->state('admin')->create();
```

Comment:
It looks like it's `states()` in the documentation.

```php
$users = factory(App\User::class, 5)->states('delinquent')->make();
```

Reply:
fixed